src/DispatchPackedConversions.jl

```diff
@@ -45,7 +45,7 @@ function convert(
   ccw = prepgenericconvolution(DFG.DFGVariable[], usrfnc, multihypo=mhcat, nullhypo=nh)
   ccw.certainhypo = d.certainhypo
 
-  ret = FunctionNodeData{CommonConvWrapper{typeof(usrfnc)}}(d.fncargvID, d.eliminated, d.potentialused, d.edgeIDs, Symbol(d.frommodule), ccw, d.multihypo, d.certainhypo)
+  ret = FunctionNodeData{CommonConvWrapper{typeof(usrfnc)}}( d.eliminated, d.potentialused, d.edgeIDs, ccw, d.multihypo, d.certainhypo, d.solveInProgress)
```

Comment:
@dehann should solveInProgress be reset to 0 or also unpacked?

Reply:
should be unpacked

Reply:
idea is to get concurrency through the DB, so all concurrency flags need to be maintained and should not be reset locally.

Reply:
"maintained" being a polling (or pull) model, not a notification/push model.